Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed.py

```diff
@@ -326,9 +343,10 @@ def main():
         'test-module': test_module_command,
         'sixgill-get-indicators': get_indicators_command
     }
+    confidence = arg_to_number(demisto.params().get("confidence"), "confidence", required=False)
```

Comment:
that is good, however if confidence = 'all', it won't fail on `ValueError`?

Reply:
I am verifying, will update you.

Reply:
Tested and fixed. Please check. Let me know if anything else is required